tests/30-main.spec.js

```diff
@@ -254,4 +255,450 @@ describe('main', () => {
       credential, documentLoader, verifyRevocationListCredential: false});
     result.verified.should.equal(false);
   });
+
+  it('should fail to verify status on missing credential param', async () => {
+    let err;
+    let result;
+    try {
+      result = await checkStatus({
+        documentLoader, verifyRevocationListCredential: false});
+      result.verified.should.equal(false);
+    } catch(e) {
+      err = e;
+    }
+    should.not.exist(err);
+    should.exist(result);
+    result.should.be.an('object');
+    result.should.have.property('verified');
+    result.verified.should.be.a('boolean');
+    result.verified.should.be.false;
+    result.should.have.property('error');
+    result.error.should.be.instanceof(TypeError);
+    result.error.message.should.contain('"credential" must be an object');
+  });
+
+  it('should fail to verify if credential is not an object for ' +
+   'statusTypeMatches"', async () => {
+    let err;
+    let result;
+    try {
+      result = statusTypeMatches('{credential}');
+    } catch(e) {
+      err = e;
+    }
+    should.exist(err);
+    should.not.exist(result);
+    err.should.be.instanceof(TypeError);
+    err.message.should.contain('"credential" must be an object');
+  });
+
+  it('should fail to verify if credential is not an object for ' +
+    '"assertRevocationList2020Context"', async () => {
+    let err;
+    let result;
+    try {
+      result = assertRevocationList2020Context('{credential}');
```

Comment:
So from here down the literal string `'{credential}'` is being passed in instead of passing in an object with the named param `credential` and some invalid object. These tests should be tweaked to better reflect that they are testing what their text says.